packages/config-transformer/tests/Finder/ConfigFileFinder/ConfigFileFinderTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\ConfigTransformer\Tests\Finder\ConfigFileFinder;
+
+use Symplify\ConfigTransformer\Finder\ConfigFileFinder;
+use Symplify\ConfigTransformer\Kernel\ConfigTransformerKernel;
+use Symplify\ConfigTransformer\ValueObject\Configuration;
+use Symplify\PackageBuilder\Testing\AbstractKernelTestCase;
+
+final class ConfigFileFinderTest extends AbstractKernelTestCase
+{
+    private ConfigFileFinder $configFileFinder;
+
+    protected function setUp(): void
+    {
+        $this->bootKernel(ConfigTransformerKernel::class);
+
+        $this->configFileFinder = $this->getService(ConfigFileFinder::class);
+    }
+
+    public function test(): void
+    {
+        $configuration = new Configuration([__DIR__ . '/Fixture'], true);
+
+        $fileInfos = $this->configFileFinder->findFileInfos($configuration);
```

Comment:
The test actually must be in the "finder" level.

The convert test only test the converter. It accepts any file, because the it is not responsible for the file finding/filtering.